src/ray/raylet/agent_manager.cc

```diff
@@ -135,8 +135,15 @@ void AgentManager::StartAgent() {
     RAY_LOG(ERROR)
         << "The raylet exited immediately because the Ray agent failed. "
            "The raylet fate shares with the agent. This can happen because the "
-           "Ray agent was unexpectedly killed or failed. See "
-           "`dashboard_agent.log` for the root cause.";
+           "Ray agent was unexpectedly killed or failed. Agent can fail when\n"
+           "- The version of `grpcio` is not respected. Agent can segfault. "
```

Comment:
nir: what do you mean by not respected here?

Reply:
```
           "- The version of `grpcio` doesn't follow Ray's requirement. "
           "Agent can segfault with the incorrect `grpcio` version. "
           "Check the grpcio version `pip freeze | grep grpcio`.\n"
```
Changed it to this sentence (I believe it is less confusing)